packages/build-ui/src/pages/collection/filter_sidebar.rs

```diff
@@ -0,0 +1,109 @@
+use bdk::prelude::*;
+
+use crate::pages::collection::FilterDropdown;
+
+
+
+#[component]
+pub fn FilterSidebar() -> Element {
+    rsx! {
+        div { class: "w-64 bg-[#000000] border-r border-[#333] p-4 flex flex-col gap-6",
+            // Artist Section
+            div { class: "flex flex-col gap-2",
+                div { class: "text-sm font-semibold", "Artist" }
+                div { class: "relative",
+                    input {
+                        class: "w-full bg-[#222] border border-[#333] text-white text-sm rounded-none p-2 pl-8",
```

Comment:
```suggestion
                        class: "w-full bg-xxxx border border-xxx text-white text-sm rounded-none p-2 pl-8",
```

define colors to `theme.css` first.
- https://github.com/biyard/ratel/blob/dev/packages/main-ui/public/theme.css
- https://github.com/biyard/ratel/blob/b1ba98d4f1f16ad29993d5edfbfc61bbc9f7fa03/packages/main-ui/src/components/button/primary_button.rs#L16